client-linuxapp/src/serviceinfo.rs

```diff
@@ -52,6 +52,23 @@ fn set_perm_mode(path: &Path, mode: u32) -> Result<()> {
     Ok(())
 }
 
+fn create_user(user: &str) -> Result<()> {
+    let user_info = passwd::Passwd::from_name(user);
+    if user_info.is_none() {
+        log::info!("Creating user: {}", user);
+        Command::new("useradd")
+            .arg("-m")
+            .arg(user)
+            .spawn()
+            .context("Error spawning new user")?
```

Comment:
this context doesn't really help as this isn't "spawning new user" but the command itself to create a user